src/stable/test/benchmarks/index.ts

```diff
@@ -46,8 +47,15 @@ async function updateBenchmarksForCanisters(
     return canisterNames.reduce(async (accPromise, canisterName) => {
         const acc = await accPromise;
         const canisterId = getCanisterId(canisterName);
-        const actor = await createActor(canisterId, whoami());
-        const currentBenchmarks = await actor._azle_get_benchmarks();
+        const currentBenchmarks = JSON.parse(
+            execSync(
+                `dfx canister call ${canisterId} _azle_get_benchmarks --output json --candid ${AZLE_ROOT}/src/stable/test/benchmarks/_azle_get_benchmarks.did`,
```

Comment:
We need to make sure this will work outside of the Azle repository for developers

Reply:
Do a release candidate and check that this works on that released version

Reply:
done